edx_proctoring/static/proctoring/js/exam_action_handler.js

```diff
@@ -49,11 +49,29 @@ var edx = edx || {};
        });
   };
 
+  function createWorker(url) {
+    var blob, blobBuilder, blobUrl, url;
+    var importScript = "importScripts('" + location.origin + url + "');";
+
+    try {
+      blob = new Blob([importScript], { "type": 'application/javascript' });
+    } catch (e) {
+      //older browsers do not support the Blob constructor, so fallback to the deprecated BlobBuilder
```

Comment:
On closer review it looks like Blob has plenty of support for our needs: https://developer.mozilla.org/en-US/docs/Web/API/Blob/Blob#Browser_compatibility c.f. https://edx.readthedocs.io/projects/edx-guide-for-students/en/latest/front_matter/browsers.html

We should be able to do without any try/catches, as it turns out